Automattic/Gravatar-SDK-Android
# Quick Editor: More options button in avatar picker

Closes #411 

### Description

Part of the milestone `2.1.0` is to add the possibility to modify the alternative text, delete an avatar, or define its rating.

The common entry point for those three options is a button in each avatar in the avatar picker.  

![Image](https://github.com/user-attachments/assets/96a6c095-2bf8-4f33-9727-0d9aec06f6ac)

This PR implements the more options button to allow the parallelized implementation of those three features.

I've added an "Alt Text" button for the sake of define all the architecture and how it should look, however it doesn't execute any action under the hood.

I've generalized our previous `Popup` menu, as this one is pretty similar, so we can keep all the code centralized.

For now, the popup is aligned with `CenterHorizontally`; we can iterate on this if we think that's not the best option.

<img width="250" alt="image" src="https://github.com/user-attachments/assets/cadf625e-439d-441f-ae08-c84b7a9c6797">

### Testing Steps

1. Open the QE in the demo-app
2. Verify you can see the more options button in each of the avatars
3. Click any of the more options buttons and verify the Popup with the menu is opened
4. Smoke test the upload new avatar popup menu
